pkg/scheduler/internal/podinfo/utils.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package util
+package podinfo
```

Comment:
Please rename this file to `podinfo.go`.

Reply:
utils.go -> podinfo.go
utils_test.go -> podinfo_test.go

Is this the expected rename ?

Reply:
yes, thanks